.github/workflows/ci.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Install
-        run: yarn install --frozen-lockfile
+        run: yarn install --frozen-lockfile --ignore-engines
```

Comment:
I don't think the changes to this file are necessary now that I merged #136. Sorry about that.